MIDAS/lib/TCAL9539/TCAL9539.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include<Arduino.h>
+
+struct GpioAddress {
+    int gpio_id;
+    int pin_id;
+};
+
+constexpr GpioAddress BNO85_RESET = {.gpio_id = 0, .pin_id = 0};
```

Comment:
I get that PinAddress and GPIOAddress are different for encapsulation or something along those lines, but can't we just unify them and use private fields for encapsulation, so we don't have to call decode to calculate the same result every single time?

Reply:
Having the separate was mainly so GpioAddress could be constexpr. I realize I can just make the constructor of gpio address be constexpr and do the decoding though.